www/src/pages/en/deployment/docker.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Docker
+description: Deployment with Docker
+layout: ../../../layouts/mainLayout.astro
+---
+
+You can containerize this stack and deploy it as a single container using Docker, or as a part of a group of containers using docker-compose.
+
+Please note that Next.js requires a different process for buildtime (available in the frontend, prefixed by `NEXT_PUBLIC`) and runtime environment variables. In this demo we are using two variables, `NEXT_PUBLIC_FOO` and `BAR`. Pay attention to their positions in the `Dockerfile`, command-line arguments, and `docker-compose.yml`.
+
+## Docker
+
+1. In your [next.config.mjs](./next.config.mjs), add the `output: "standalone"` option to your config.
```

Comment:
Resolve link here

Reply:
Where should these resolve to? The [file in the repo](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs)?

Reply:
I think that makes sense?